### PR TITLE
Update main.js (Object destructuring)

### PR DESCRIPTION
**Pull request by miko-github**
# Destructruing Electron Modules

Hi, :wave:
I did only **destructuring** object from electron module :heart: 

i commented prev code, then, if you want to remove, remove comment and use :bulb: 

[**mozila docs**](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring)

## thank you for this awesome browser :tada:

### CODE REVIEW

File: main/main.js [hunk 1]
```diff
@@ -1,15 +1,32 @@
 const electron = require('electron')
```
Comment (PalmerAL) [inline main/main.js:1]: This should also be part of the object destructuring

**Comment by PalmerAL** [comment]
I think this is fine, can you remove the old code?

File: main/main.js (hunk 1 above)
Comment (miko-github) [reply main/main.js:1]: # Hi PalmerAL :wave: 
exactly  yes, but i think, **may** that need to use **electron** module :wink: 

## **thanks for your review :heart:**

**Comment by miko-github** [comment]
> I think this is fine, can you remove the old code?

so, we need to accept of owner of this repo, if he accepted, he can remove or he can save :hugs:

File: main/main.js (hunk 1 above)
Comment (PalmerAL) [reply main/main.js:1]: Oh, nevermind, I misread what was happening here.

**Comment by PalmerAL** [comment]
Thanks.

**Pull request merged**